change/@microsoft-fast-foundation-28b13446-5fdd-4c7d-a077-32ec0c284e8a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix: it prevents to increment/decrement the value of a readonly or disabled number-field",
```

Comment:
could say "it prevents incrementing/decrementing the value ..."

Reply:
Sure